digdag-core/src/main/java/io/digdag/core/database/migrate/Migration_20160926123456_AddDisabledColumnToSchedules.java

```diff
@@ -0,0 +1,13 @@
+package io.digdag.core.database.migrate;
+
+import org.skife.jdbi.v2.Handle;
+
+public class Migration_20160926123456_AddDisabledColumnToSchedules
```

Comment:
`timestamp` type (`timestamp with time zone` on postgresql) seems better in this case.

I used `bigint` for session_time and next_run_time because I wanted to avoid confusion of different handling of timestamp precision depending on databases (nano, micro, or seconds). Precision of schedules and session_time are fixed to seconds. But `foobar_at` columns should have more precision.

```
        handle.update("alter table schedules add column disabled_at bigint");
```